lib/tasks/send_presentation_notifications.rake

```diff
@@ -0,0 +1,5 @@
+require 'rake'
+
+task send_presentation_notifications: :environment do
+  EventNotificationJob.perform
```

Comment:
So it took me until now to realize this, but our job doesnt take any arguments. Which means we can't take advantage of splitting our event notifications out into multiple processes to make them faster.

I propose making EventNotificationJob#perform accept an event as an argument. 
It will be the rake tasks responsibility to find all events that need to be notified, and then initialize a new job for every event.

That way when we implement our background processing. The queue can run multiple jobs at the same time, instead of having to notify every event from within the same job